R/accounts.R

```diff
@@ -303,8 +325,13 @@ accountInfo <- function(name = NULL, server = NULL) {
   info <- as.list(accountDcf)
   # remove all whitespace from private key
   if (!is.null(info$private_key)) {
-    info$private_key <- gsub("[[:space:]]", "", info$private_key)
+    info$private_key <- secret(gsub("[[:space:]]", "", info$private_key))
```

Comment:
Does `rsconnect::accounts` need the same adjustment?

This adjusts responses from the package API and should be included in NEWS.md.

Reply:
`accounts()` just includes `name` and `server` — did you mean `servers()`?

Reply:
Probably, yes!

Reply:
I'm not sure that the `certificates` field in `servers()` is actually secret, but it is very long, so redacting it is a service to the user.